introduction.md

```diff
@@ -1,3 +1,10 @@
 Hi, my name is  Reny Alvarado
 My favorite Code School path is Ruby
 
+Deadly Skills
+=============
+
+* Git
+* Javascript
+* Java
+x
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?